rc/filetype/nginx.kak

```diff
@@ -0,0 +1,30 @@
+hook global BufCreate .*\.(conf) %{
+    set-option buffer filetype nginx
```

Comment:
this has the same problem of false positives.
Probably better to restrict this to files under /etc/nginx/

Reply:
that's what I thought after mawww's comment with the other conf files.